pkg/controller/core/clusterqueue_controller.go

```diff
@@ -157,8 +157,18 @@ func (r *ClusterQueueReconciler) notifyWatchers(oldCQ, newCQ *kueue.ClusterQueue
 	}
 }
 
-func (r *ClusterQueueReconciler) NotifyResourceFlavorUpdate(rf *kueue.ResourceFlavor) {
-	r.rfUpdateCh <- event.GenericEvent{Object: rf}
+func (r *ClusterQueueReconciler) NotifyResourceFlavorUpdate(oldRF, newRF *kueue.ResourceFlavor) {
```

Comment:
Add a comment that Updates are skipped because they don't affect the readiness of the ClusterQueue